docs/guides/getting-started/setup/integrate.mdx

```diff
@@ -0,0 +1,113 @@
+---
+description: If you already have an existing web project, this guide will walk you through integrating Tauri into your project
+pagination_next: guides/development/development-cycle
+sidebar_position: 3
+sidebar_custom_props:
+  doc_card_image: /img/guides/getting-started/setup/html5-light.svg
+  doc_card_image_dark: /img/guides/getting-started/setup/html5-dark.svg
+---
+
+import Commands from './_fragments/_commands.mdx'
+import TauriInit from './_fragments/_tauri-init.mdx'
+import { InstallTauriCli, InstallTauriApi } from '@theme/Command'
+
+# Integrate into Existing Project
+
+If you already have an existing web project, this guide will walk you through integrating Tauri into your project, whether it is Node.js-based (like [Svelte], [React], [Vue], or [Angular]) or Rust-based (like [Yew] or [Dominator]).
+
+:::info
+
+Before we continue, make sure you have completed the [prerequisites] to have a working development environment.
+
+:::
+
+Although Tauri is compatiable with nearly any frontend framework, we'll use examples with a [React] project throughout this guide. We'll assuming you're starting off with a project structure similar to this:
+
+```
+.
+│── package.json
+│── public
+│   ╰── index.html
+╰── src
+    │── App.css
+    │── App.js
+    │── index.css
+    ╰── index.js
+```
+
+## Create the Rust Project
+
+<TauriInit
+  destDirExplination='Use the path where your compiled web assets will be located. This may be something like "../dist" or "../build".'
+  devPathExplination='This is the path or address your frontend is served at during development. If servering from localhost this could be "https://localhost:3000". This can also be a directory similar to the above question.'
+  beforeDevCommandExplination='Use "npm run dev" for this value (be sure to adapt to use your package manager of choice).'
+  beforeBuildCommandExplination='Use "npm run build" for this value (be sure to adapt to use your package manager of choice).'
+/>
+
+And that's it, you have now added Tauri to your existing project and you should see a `src-tauri` directory that looks something like this:
+
+```diff {9-14}
+│── package.json
+│── public
+│   ╰── index.html
+│── src
+│   │── App.css
+│   │── App.js
+│   │── index.css
+│   ╰── index.js
+╰── src-tauri
+    │── Cargo.toml
+    │── build.rs
+    │── icons
+    │── src
+    ╰── tauri.conf.json
+```
+
+## Add JavaScript API Package (Optional)
```

Comment:
we have optional here but later we require it for the commands section. we need to make two flavors of commands, one using the globalTauri and one with the npm package

Reply:
How's this now?

Reply:
I was thinking more of a toggle that changes the code in the invoke example to the global tauri style.